scripts/src/main/resources/scripts/functions

```diff
@@ -167,6 +167,99 @@ function doEchoInteraction() {
   echo -e "\033[96m${*}\033[39m"
 }
 
+# $1: version_start
+# $2: version_end
+# $3: version
+# returns 0 if version_start <= version <= version_end
+function doCheckVersionRange() {
+  local version1="${1}"
+  local version2="${2}"
+  local version="${3}"
+  doVersionCompare "${version}" "${version1}"
+  check1="${?}"
+  doVersionCompare "${version2}" "${version}"
+  check2="${?}"
+  if [ "${check1}" == "2" ] || [ "${check2}" == "2" ] 
+  then
+    return 1
+  else 
+    return 0
+  fi
+}
+
+# $1: software
+# $2: version
+# return 1 if version is found in tool's security file
+function doCheckSoftwareSecurityVersion() {
+  local software="${1}"
+  local version="${2}"
+  local edition="${3}"
+  local security_file
+  local check="0"
+  local line=""
+  local myfirst
+  local mylast
+  if [ -d "${DEVON_IDE_HOME}/ide-urls" ]
+  then
+    if [ -z "${edition}" ]
+    then
+      edition="${software}"
+    else
+      doTranslateMirrorOptionsToUrlOptions
+    fi
+    security_file="${DEVON_IDE_HOME}/ide-urls/${software}/${edition}/security"
+  else
+    security_file="${DEVON_IDE_HOME}/mirrors/${software}/security"
```

Comment:
actually `mirrors` is dead. When we add this new feature that will most probably be released together with #941 we can simply assume that the new approach is already in place. Why should we then release legacy code handling `mirrors`?
Besides, we need to clarify the folder name. It can be defined as `ide-urls` but then we have to also specify this in the epic or the story to implement the logic that is cloning this new repo into DEVON_IDE_HOME. I can do that so things will fit together...

Reply:
Part 2: If I had looked in the epic I would of course have specified the desired directory name urls in the git clone. This will of course be corrected immediately.

Part 1: This was intended for cases in which a devon ide update scripts is started, but this command does not include a git clone repo. But it's actually not needed, since the functions are only runs if a security file is found.